packages/fxa-settings/src/pages/Signin/Signin/index.tsx

```diff
@@ -0,0 +1,248 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactElement, useCallback, useState } from 'react';
+import { usePageViewEvent } from '../../../lib/metrics';
+import { useFtlMsgResolver } from '../../../models';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { RouteComponentProps } from '@reach/router';
+import InputPassword from '../../../components/InputPassword';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { ReactComponent as PocketLogo } from './pocket.svg';
+
+type SigninProps = {
+  email: string;
+  isPasswordNeeded: boolean;
+  isPocketClient: boolean;
```

Comment:
Instead of passing `isPocketClient` as a prop, can we just check against `serviceName`?

We should probably start a type or enum for `serviceName`, and we can check against that value directly, e.g.:

```
enum ServiceName {
  Settings = 'account settings',
  Pocket = 'Pocket',
  // etc.
}

...
const isPocketClient = serviceName === ServiceName.Pocket;
```

You can also set the default `serviceName` to the enum too, and you can export this and use it in tests/Storybook. This will probably be useful in other files so it might should go in `lib` somewhere if that's already the case.